src/D2L.Bmx/UpdateHandler.cs

```diff
@@ -87,6 +87,13 @@ internal class UpdateHandler( IGitHubClient github, IVersionProvider versionProv
 		}
 
 		Console.WriteLine( $"BMX updated to v{latestVersion} successfully!" );
+
+		try {
+			Directory.Delete( workspaceDir, recursive: true );
```

Comment:
Is there any point in deleting stuff moved to ``/tmp`` explicitly like this? Most distros have some sort of cleanup that generally takes care of this.

Reply:
At least Windows doesn't auto delete temp files by default.